kubeflow/trainer/backends/localprocess/backend.py

```
@@ -0,0 +1,258 @@
+
+        for _step in _job[0].steps:
+            if not want_all_steps and _step.step_name != step:
+                continue
+            # Flatten the generator and pass through flags so it behaves as expected
+            # (adjust args if stream_logs has different signature)
+            yield from _step.job.logs(follow=follow)
+
+    def wait_for_job_status(
+        self,
+        name: str,
+        status: set[str] = {constants.TRAINJOB_COMPLETE},
```

Ruff (0.12.2)

Unused method argument: `status`

(ARG002)